spec/view-registry-spec.coffee

```diff
@@ -26,6 +26,14 @@ describe "ViewRegistry", ->
         expect(node.textContent).toBe "Hello"
         expect(node.spacePenView).toBe view
 
+    describe "when passed an object with an element property", ->
+      it "returns the element property if it's an instance of HTMLElement", ->
```

Comment:
Not in love with the naming here, so I'm open for suggestions to improvements. /cc @atom/feedback